db/db-challenge2/ddl.sql

```diff
@@ -0,0 +1,57 @@
+CREATE TABLE users (
+    user_id INTEGER(11) PRIMARY KEY AUTO_INCREMENT,
+    username VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL UNIQUE,
+    password VARCHAR(100) NOT NULL,
+    introduction VARCHAR(1000),
+    mobile_phone VARCHAR(13) UNIQUE,
+    business_phone VARCHAR(13),
+    is_deleted TINYINT(1) DEFAULT 0 NOT NULL,
+    created_at DATETIME NOT NULL,
+    modified_at DATETIME NOT NULL
+)
```

Comment:
DDL(ddl.sql)は正常に実行できることを確認してください～
各 create table 文の最後にセミコロン;を入れてください
```suggestion
);
```